src/client/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -85,6 +87,15 @@ public async Task<IEnumerable<IAccount>> GetAccountsAsync()
         {
             RequestContext requestContext = CreateRequestContext(Guid.NewGuid());
             IEnumerable<IAccount> accounts = Enumerable.Empty<IAccount>();
+
+#if ANDROID
+            if (AppConfig.IsBrokerEnabled)
+            {
+                var broker = ServiceBundle.PlatformProxy.CreateBroker(null);
+                accounts = await broker.GetAccountsAsync(AppConfig.ClientId).ConfigureAwait(false);
```

Comment:
>GetAccountsAsync(AppConfig.ClientId) [](start = 40, length = 36)

question: GetAccountsAsync returns the accounts based on the clienttId here? shouldn't it be based on the user?

Reply:
no, there is no user specified as a parameter because this flow was designed to get all users from the broker. The broker requires a client ID when making a request to get all accounts because it separates accounts by client ID.